proposals/fleet-desktop-token-rotation.md

```diff
@@ -0,0 +1,64 @@
+# Token Rotation for Fleet Desktop
+
+This file is based on the original proposal described at [#6348](https://github.com/fleetdm/fleet/issues/6348) modified based on a few lessons we learned and the new communication channel between Orbit and the Fleet server introduced in https://github.com/fleetdm/fleet/issues/6851
+
+**Compatibility**
+
+|                | Fleet Desktop < v1.0.0            | Fleet Destkop >= v1.0.0 |
+| -------------- | --------------------------------- | ----------------------- |
+| Server < 4.21  | OK/Rotation disabled              | OK/Rotation disabled    |
```

Comment:
To over-communicate: To support `Server < 4.21` with `Fleet Destkop >= v1.0.0`, we'll need to keep the `orbit_info` table extension in Orbit up and running.

Reply:
+1 this is noted in the "Orbit > Compatibility" section below. I believe we'd still need that extension running to support live queries, right?

Reply:
True. It's a handy table. We'll be adding more "useful" Orbit fields to that table soon.